client/value.go

```diff
@@ -18,13 +18,15 @@ import (
 type FieldValue struct {
 	t       CType
 	value   any
+	kind    FieldKind
```

Comment:
question:  This to have no use.  Why has it been added?

Reply:
changed